### PR TITLE
RHEL6 STIG References part 2

### PR DESCRIPTION
**Pull request by thenefield**
- Removed double-spacing introduced from the previous commit.
- Executed the stig_refs.sh script with its corrections, which brought forth additional changes.

### CODE REVIEW

File: RHEL/6/input/system/software/integrity.xml [hunk 1]
```diff
@@ -176,9 +176,9 @@ have file hashes different from what is expected by the RPM database.
 The hashes of important files like system executables should match the
 information given by the RPM database. Executables with erroneous hashes could
 be a sign of nefarious activity on the system.</rationale>
-<ident cce="27223-7"  stig="RHEL-06-000519" />
+<ident cce="27223-7" stig="RHEL-06-000519" />
 <oval id="rpm_verify_hashes" />
-<ref nist="CM-6(d),SI-7" disa="1496" />
+<ref nist="CM-6(d),SI-7" disa="366" />
```
Comment (shawndwells) [inline RHEL/6/input/system/software/integrity.xml]: there's something funky happening in the script. This rule maps to CCI-001496, which is RHEL-06-00281 in DISA's content (not RHEL-06-000519).

Comment (thenefield) [inline RHEL/6/input/system/software/integrity.xml]: So here is what's happening. The script utilizes
input/auxiliary/stig_overlay.xml as a sort of legend mapping rule id to
stig id. The input/auxiliary/stig_overlay.xml file currently
references RHEL-06-000519
for rule id rpm_verify_hashes. Its doing what its suppose to. Perhaps in
this case, the input/auxiliary/stig_overlay.xml document should be updated
to reflect the correct STIG requirement for that rule. Looking at the
RHEL-06-00281
stig id in input/auxiliary/stig_overlay.xml shows no rule is currently
associated with it.

**Comment by redhatrises** [comment]
@thenefield please fix the merge conflicts and issues before this can be merged. There are also files that are missing space between ```"disa``` in various files.

**Comment by isimluk** [comment]
What was the build failure? I have no permissions to see it.

**Comment by iankko** [comment]
@isimluk 
> What was the build failure? I have no permissions to see it.

Unless I am wrong just Jenkins administrator can see the failures (if you refer e.g. to access to:
  http://jenkins.ssgproject.org:8080/job/scap-security-guide/324/ )

Also assuming that @redhatrises in: 
  [1] https://github.com/OpenSCAP/scap-security-guide/pull/461#issuecomment-87758409

commented on the conflicts just because they tried to merge manually in local Git clone (Gabe please correct me if I am wrong and you are able to see the link [1] ).

Which brings us to the point - @shawndwells Shawn, would it be possible to add some kind of guest Jenkins user to have overall read privilege on the completed jobs so people could see the failures without the need to have local Git repo clone ?

Thanks, Jan.

**Comment by redhatrises** [comment]
@iankko you are correct, and I cannot see the link. It was a manual checkout.

@isimluk I am assuming that this PR is failing because of Parser Errors (missing required whitespaces) that are added by this PR as well as `make validate` failing because of those Parser Errors.

**Comment by thenefield** [comment]
@redhatrises The missing white spaces have been corrected and 'make validate' produces no errors.

**Comment by ybznek** [comment]
Sorry for inconvenience, maybe I have caused "failed" tests of this PR by rerunning of them.

**Comment by iankko** [comment]
Taking this one. Will create updated version post 0.1.30 release (to settle down in 0.1.310.

**Comment by mpreisler** [comment]
@iankko Some variant of this has gotten into 0.1.30 if I am not mistaken. Could you please confirm and close this?

**Comment by iankko** [comment]
The double space in selected ```<ident>``` elements for RHEL-6 content is now replaced by:
* https://github.com/OpenSCAP/scap-security-guide/pull/1334

As for the ```disa=``` element changes for selected RHEL-6 rules (which were also part of this PR), I am not gonna to change those (not familiar with the way CCI ID to disa ID mapping is created. But assuming some ID would be wrong / inappropriate, it would have been changed from the time this PR was created already).

Closing in favour of https://github.com/OpenSCAP/scap-security-guide/pull/1334

**Pull request closed without merging**